SentinelOne/sentinelone_module/singularity/client.py

```diff
@@ -0,0 +1,129 @@
+from contextlib import asynccontextmanager
+from typing import Any, AsyncGenerator
+from urllib.parse import urljoin
```

Comment:
It's better to use `posixpath.join` than `urllib.parse.urljoin) (see [this PR](https://github.com/SEKOIA-IO/sekoia-automation-sdk/pull/139))
```suggestion
from posixpath import join as urljoin
```